tests/test_selection/test_drop_features.py

```diff
@@ -26,6 +26,13 @@ def test_drop_1_variable(df_vartypes):
     assert type(X) == pd.DataFrame
     pd.testing.assert_frame_equal(X, df)
 
+    # checking features_to_drop has been correctly cast to list of str internally
+    transformer = DropFeatures(features_to_drop='Marks')
+    X = transformer.fit_transform(df_vartypes)
+    assert X.shape == (4, 4)
+    assert transformer.features_to_drop == "Marks"
+    assert type(X) == pd.DataFrame
+
```

Comment:
could we please add an assert to compare the result of the transformation with what we expect from it? see line 27